### DISCUSSION TITLE
Weekly Newsletter - Recently Added Content

### DISCUSSION THREAD

**Discussion by GD-L** [sct/overseerr#2003] [Feature Requests]
Recently moved from Ombi to Overseer. One of the things my users liked having was the weekly emailed newsletter of "Recently added content"
I think this would be a very valuable addition to Overseer.

**Comment by 5H3RM** [comment]
While this would be a nice idea I think it would be a low priority as this can be done with other tools.

**Comment by epheterson** [comment]
Would also like to see this, I setup Tautulli exclusively to send a newsletter (as Plex has adapted many of the other stats/features making Tautulli largely redundant). If Overseer could send newsletters and ideally also have a landing page (no sign-in required) of recent content, there's no need for Tautulli or other tools.

Also Tautulli doesn't support allowing users to unsubscribe, so it's possible Overseer could create a better offering than other tools.

**Comment by bruvv** [comment]
I am in the same boat, this is keeping me from switching from ombi. This would be awesome addition.

**Comment by Thunder7ga** [comment]
Currently doing this with Tautulli, and my users love it.  I'd love to see this feature included here instead.  I used Ombi for a bit, and was about to switch my weekly newsletter to it, but then decided to move away from Ombi.
